frontend/src/services/updateData.js

```diff
@@ -0,0 +1,22 @@
+const baseUrl = 'http://urbanplants.local/plant'
+
+export default async function postData(data, id) {
```

Comment:
same here: postPlant or updatePlant
And is there a reason, you use different names for file and function?